create-plugin-update/action.yml

```diff
@@ -37,17 +38,16 @@ runs:
     - name: Get version from .config/.cprc.json
       id: get-config-version
       run: |
-        CONFIG_VERSION=$(jq -r '.version' .config/.cprc.json)
-        echo "CONFIG_VERSION=${CONFIG_VERSION}" >> "$GITHUB_ENV"
+        echo "CONFIG_VERSION=$(jq -r '.version' .config/.cprc.json)" | tee -a "$GITHUB_ENV"
       shell: bash
 
     - name: Compare versions
       id: compare-versions
       run: |
         if [ "$LATEST_VERSION" != "$CONFIG_VERSION" ]; then
-          echo "update_needed=true" >> $GITHUB_OUTPUT
+          echo "update_needed=true" | tee -a "$GITHUB_OUTPUT"
```

Comment:
For my own understanding, what is the benefit of using `tee` over `>>`?

Reply:
It just writes the line to stdout as well as to the file, which makes it a bit easier to see what was going on -- happy to revert them if you'd rather the output be cleaner.

Reply:
nit: ok, personally I'd vote to put them back as they are but I'm not gonna block it.